tests/unit_tests/data_steward/resources_test.py

```diff
@@ -65,3 +68,76 @@ def test_cdm_schemas(self):
             result_vocab_tables,
             msg='Vocabulary tables %s should not be in result of cdm_schemas()'
             % result_vocab_tables)
+
+    def test_fields_for(self):
+        """
+        Testing that fields for works as expected with sub-directory structures.
+        """
+        # preconditions
+
+        # test
+        actual_fields = resources.fields_for('person')
```

Comment:
This is the original function signature.  I wanted to make sure the new implementation didn't break existing function calls (there are quite a few throughout the code base).  I used the person table because it is a prominent omop table and is used extensively. Also, it's `post_deid_person.json` counterpart can cause trouble.  So I want to make sure the `fields_for` function is finding the correct file.
Yes, I explicitly open the person file I expect `fields_for` to open.  This is a file that exists as part of our repository, so I can expect it to be present.  If that file goes missing, it would mean something has changed structurally within our code and the failing test would alert us that whatever change was made is a breaking change.  I compare that json object to the json object I get from `fields_for` to make sure it's opening the correct file.

Reply:
This makes sense.